azerothcore/azerothcore-wotlk
# fix: Loch Modan multiple ore spawn

## Changes Proposed:
- Adds Tin and Silver ores to the pools, making it so that only one can spawn at the time

## Issues Addressed:
- Closes https://github.com/chromiecraft/chromiecraft/issues/3661

## SOURCE:
The changes proposed on this PR are just a fix to the current situation, where multiple ores spawn at the same time in Loch Modan. I haven't verified that the resulting behavior after applying this matches anything that can be considered blizzlike, but it does get rid of the multiple ores problem.

## Tests Performed:
This PR has been:


## How to Test the Changes:

1. Run SQL commands
2. Verify that 333 records were inserted
3. Verify in-game that no multiple ores spawn on a single point in Loch Modan

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
